c2corg_ui/static/partials/trackdownload.html

```diff
@@ -0,0 +1,2 @@
+<button class="btn btn-primary" ng-click="trackCtrl.downloadGpx()">GPX</button>
```

Comment:
Links are not OK on mobile device. And it has to be quite obvious. I prefer a button.

I agree that improvements such as styling (color, icon, size) or placement (perhaps it's not the best place to put the buttons?) could be done. But I think it can be done later. I prefer to merge as is since the PR looks good to both of you aside of this :)

I will remove the extra @export and merge when Travis is OK. Thanks for the review/comments!